.gitignore

```diff
@@ -59,4 +60,4 @@ lib/*/lib/*.lib
 lib/qtscript-bytearray/*.cc
 
 *.vscode
-compile_commands.json
\ No newline at end of file
+compile_commands.json
```

Comment:
I'll fix this as soon as I'm okay with eclipse..

Reply:
It would be ok for me to add the eclipse files to the .gitignore 
This just tells everyone that we do not want them in our code base which is true.